src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1653,6 +1664,8 @@ else if (environmentName != null) {
                 this.contentAccessManager.removeContentAccessCert(toUpdate);
             }
 
+            contentAccessManager.syncOwnerLastContentUpdate(owner);
```

Comment:
I think this was my fault mentioning this in the card; you're right it's not needed, since we're already deleting the consumer's cert it will be regenerated anyway

Reply:
Removed.